libdecor/libdecor-0.1.1.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "libdecor",
+  "buildsystem": "meson",
```

Comment:
Missing `"cleanup"` for all development files installed.

Reply:
Ah, I don't think I know the specific devel files I'll need to clean up >>

Would I need to go into the flatpak environment and see what I need to get rid of?

Reply:
Okay, I *think* I've found the stuff I need to clean up? Would adding these be fine? https://github.com/flathub/org.nongnu.enigma/blob/62d92b83efec2c3c1bc3fd217f53e204fb977993/org.nongnu.enigma.yml#L42